index.js

```diff
@@ -388,7 +389,11 @@ async function connect(client, requeue, gamemode) {
 		});
 	})
 	ClientSocket.on("CPacketMessage", packet => {
-		if (packet.text) client.write('chat', {message: JSON.stringify({text: translateText(packet.text)})});
+		if (packet.text) {
+			client.write('chat', {
+				message: JSON.stringify({text: translateText(packet.text)})
+			});
+		}
```

Comment:
```suggestion
		if (packet.text) client.write('chat', {message: JSON.stringify({text: translateText(packet.text)})});
```
revert the formatter v5